modules/odr_core/odr_core/config.py

```diff
@@ -33,7 +33,10 @@ class Settings(BaseSettings):
     TEST_POSTGRES_DB: str
     TEST: bool = False
 
-    ROOT_DIR: str
+    ROOT_DIR: str = ""
+
+    # Models
+    MODEL_CACHE_DIR: str = f"{ROOT_DIR}/model_cache"
```

Comment:
Can we add an environment var for this instead of going from the project root?

Reply:
Added.